tsfc/fem.py

```diff
@@ -693,8 +693,17 @@ def take_singleton(xs):
     return result
 
 
-def compile_ufl(expression, interior_facet=False, point_sum=False, **kwargs):
-    context = PointSetContext(**kwargs)
+def compile_ufl(expression, context, interior_facet=False, point_sum=False):
+    """Translate a UFL expression to GEM.
+
+    :arg expression: The UFL expression to compile.
+    :arg context: translation context - either a :class:`GemPointContext`
+        or :class:`PointSetContext`
```

Comment:
I like that this `context` has been made explicit compared to the previous version.